### PR TITLE
SCMOD-5090: Resolve Dependency issue

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://autjira.microfocus.com/browse/SCMOD-5090

http://sou-jenkins2.hpeswlab.net/job/WorkerFramework/view/Developer/job/WorkerFramework~worker-framework~SCMOD-5090~CI/

### CODE REVIEW

File: worker-archetype/pom.xml [hunk 1]
```diff
@@ -37,43 +37,74 @@
...
-            <artifactId>worker-api</artifactId>
-            <version>${project.version}</version>
+            <artifactId>worker-framework</artifactId>
+            <type>pom</type>
+            <version>3.2.0-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
```
Comment (michael-bryson) [inline worker-archetype/pom.xml:52]: Should this be scope of test?

File: worker-framework/pom.xml [hunk 2]
```diff
@@ -36,28 +36,28 @@
...
             <dependency>
                 <groupId>com.github.workerframework</groupId>
-                <artifactId>util-rabbitmq</artifactId>
+                <artifactId>worker-configs</artifactId>
                 <version>3.2.0-SNAPSHOT</version>
             </dependency>
             <dependency>
                 <groupId>com.github.workerframework</groupId>
-                <artifactId>util-store</artifactId>
+                <artifactId>util-rabbitmq</artifactId>
```
Comment (michael-bryson) [inline worker-framework/pom.xml]: Incorrect alphabetical order, u comes before w.

File: worker-testing-integration/pom.xml [hunk 3]
```diff
@@ -35,22 +35,10 @@
...
-        </dependency>
-        <dependency>
-            <groupId>com.github.workerframework</groupId>
-            <artifactId>worker-core</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-testing-util</artifactId>
-            <version>3.2.0-SNAPSHOT</version>
```
Comment (michael-bryson) [inline worker-testing-integration/pom.xml:53]: Should this have scope of test?

File: worker-testing-integration/pom.xml [hunk 4]
```diff
@@ -60,31 +48,61 @@
...
-            <artifactId>woodstox-core-asl</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.9.8</version>
```
Comment (michael-bryson) [inline worker-testing-integration/pom.xml:87]: Should this have scope of test?

**Review by michael-bryson — CHANGES_REQUESTED** [review]

File: worker-framework/pom.xml (hunk 2 above)
Comment (thilagasanthosh) [reply worker-framework/pom.xml]: Fixed.

File: worker-testing-integration/pom.xml (hunk 3 above)
Comment (thilagasanthosh) [reply worker-testing-integration/pom.xml:53]: As per build log
http://sou-jenkins2.hpeswlab.net/job/WorkerFramework/job/WorkerFramework~worker-framework~develop~CI/269/consoleFull
It is not reported to have scope "test". Wherever worker-testing-integration is referred,  worker-testing-integration is declared as "test" scope. 
I tried to change to "test" scope and it is throwing compilation errors in worker-example-container/ExampleTestControllerProvider class. So it should not be "test" scope.

Also according to "mvn dependency:tree: it is "compile" scope.

File: worker-archetype/pom.xml (hunk 1 above)
Comment (thilagasanthosh) [reply worker-archetype/pom.xml:52]: Verified with "mvn dependency:tree" and scope test added.

File: worker-testing-integration/pom.xml (hunk 4 above)
Comment (thilagasanthosh) [reply worker-testing-integration/pom.xml:87]: Verified with "mvn dependency:tree" and scope test added.

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**